File: data/sources/3/1/axb.java
import java.util.*;

/** * Ilustracni priklad pro PILSPROG * @author netrvalo * @version Time&date 20:51 6.8.2007 */

public class axb {
	/** * Provede soucin n dvojic zadanych celych cisel */

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		for (int i=sc.nextInt(); i>0; i--) {
			System.out.println(sc.nextInt() * sc.nextInt());
		}
	}
}

File: data/sources/1/12/axb.c
#include <stdio.h>

/*
  Ilustracni priklad pro PILSPROG
  netrvalo
  Time&date: 20:51 6.8.2007
  Provede soucin n dvojic zadanych celych cisel
 */

int main(void) {
  int i;
  int a,b;

  scanf("%d",&i);
  for (; i>0; i--){
    scanf("%d",&a);
    scanf("%d",&b);
    printf("%d\n",a*b);
  }
  return 0;
}
